GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+
+For example, Technical governance is expected to be performed by the Maintainers of the various project areas and subprojects, as defined in the appropriate OWNERS / MAINTAINERS files.
+
+Note that if the committee is called to resolve contended decisions regularly, it is a symptom of a larger problem in the community that will need to be addressed.
+
+## Inactivity
+
+It is important for members to be and stay active to set an example and show commitment to the project. Inactivity is harmful to the project as it may lead to unexpected delays, contributor attrition, and a loss of trust in the project.
+
+* Inactivity is measured by
+  * Periods of no contributions for longer than 6 months
+  * Not having 20 contributions in the preceding 12-month window for more than 6 months
+* Consequences of being inactive include
+  * Involuntary removal or demotion
+  * Being asked to move to Emeritus status
+  * Not counting against quorum in votes
+
+## Involuntary Removal or Demotion
+
+Involuntary removal/demotion of a contributor happens when responsibilities and requirements aren't being met. This may include repeated patterns of inactivity, extended periods of inactivity, a period of failing to meet the requirements of your role, and/or a violation of the Code of Conduct. This process is important because it protects the community and its deliverables while also opening up opportunities for new contributors to step in.
+
+Inactive members can be automatically removed.
+
+Involuntary removal or demotion of Maintainers or Steering Committee members is handled through a vote by a majority of the current Steering Committee. If this pertains to a Steering Committee member, that member is not eligible to discuss or vote on the matter.
+
+## Stepping Down/Emeritus Process
+
+If and when contributors' commitment levels change, contributors can consider stepping down (moving down the contributor ladder) vs moving to emeritus status (completely stepping away from the project).
+
+Contact the Maintainers about changing to Emeritus status, or reducing your contributor level.
+
+## Contact
+
+For inquiries, please reach out to:  
+* [the governance repository](https://github.com/prometheus/governance/issues): Governance Repo Issues  
+* [prometheus-steering@googlegroups.com](mailto:prometheus-steering@googlegroups.com): Steering Committee Email
+
+## Migrating from the old governance
+
+All current Prometheus team members become members as per new governance. 
+
+The individual projects already define who is a maintainer for which specific project in their corresponding MAINTAINERS.md. People listed in these files are therefore going to be maintainers. 
+
+The existing Prometheus team is going to hold votes, as described in the new governance, to elect a Bootstrap steering committee of 7 members, where the highest voted 3 members will hold a 2 year and the next 4 members will hold a 1-year term.  
```

Comment:
If we make include "maintainers but not team members" as initial Members according to the new governance, should we also let them take part in the initial steering committee election?

Reply:
Yes.